src/executables/util_cs.c

```diff
@@ -4544,18 +4545,87 @@ flashback (UTIL_FUNCTION_ARG * arg)
 int
 memmon (UTIL_FUNCTION_ARG * arg)
 {
+#if defined(CS_MODE)
   UTIL_ARG_MAP *arg_map = arg->arg_map;
-  char *module;
+  const char *database_name;
+  char *module, *stop;
   bool transaction, show_all;
   int tran_count;
+  int module_index = MMON_MODULE_LAST + 1;
```

Comment:
```suggestion
  int module_index = MMON_MODULE_LAST;
```
Nit. MMON_MODULE_LAST itself is for this purpose.

Reply:
I think MMON_MODULE_LAST itself can work well. I change it.